components/FinalReport.js

```diff
@@ -124,6 +125,12 @@ class FinalReport extends Component {
                         this.state.exportAction(result.selectedItem.label.substring(1));
                     }}
               />
+              <VStack alignItems="center" mb={8}>
```

Comment:
This placement does not match all the other screens. Might consider adding a `ScrollView` tags and placing this outside the tag. Reference how the other pages are laid out.